client/package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "styled-components": "^5.3.10",
```

Comment:
consider fixing these versions: remove the ^ if you add any new package in the future

Reply:
Noted. Thanks